File: Iniciante/1044 - Múltiplos/JavaScript/1044.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const [a, b] = input.trim().split(/\s+/).map(Number);
  const isMultiple = checkIsMultiple(a, b);
  if (isMultiple) {
    console.log(`Sao Multiplos`);
  } else {
    console.log(`Nao sao Multiplos`);
  }
}

function checkIsMultiple(a, b) {
  const largerNumber = Math.max(a, b);
  const smallerNumber = Math.min(a, b);

  return smallerNumber !== 0 && largerNumber % smallerNumber === 0;
}

main();

File: Iniciante/1044 - Múltiplos/Python/1044.py
def main():
    a, b = map(int, input().split())
    is_multiple = check_is_multiple(a, b)
    if is_multiple:
        print("Sao Multiplos")
    else:
        print("Nao sao Multiplos")


def check_is_multiple(a, b):
    larger_number = max(a, b)
    smaller_number = min(a, b)

    return smaller_number != 0 and larger_number % smaller_number == 0


if __name__ == "__main__":
    main()
